props/configs/src/main/resources/dh-defaults.prop

```diff
@@ -29,7 +29,7 @@ TrackedFileHandleFactory.maxOpenFiles=1024
 ###### Time Zone & Calendars #####
 user.timezone=America/New_York
 timezone.aliases=/default_time_zone_aliases.csv
-Calendar.default=USNYSE
+Calendar.default=UTC
```

Comment:
I can give you the USNYSE calendar that we have which is up-to-date.  Enterprise maintains that one only.